### DISCUSSION TITLE
异步校验建议与表单错误信息提示使用疑问（2.0.0-beta.23）

### DISCUSSION THREAD

**Discussion by oswayne** [alibaba/formily#1249] [Ideas]
## 表单异步校验建议

表单异步校验当前版本的校验方式是当内容发生改变时调用异步校验方法，可以考虑在此处增加防抖。

## 表单异常信息手动设置问题

手动设置表单中字段的错误提示信息这个问题困扰了我一个周末。我在文档提供的诸多案例中并没有寻找到合适的方案。

举个例子：我在表单的提交方法（`onAutoSubmit`）中请求服务端注册用户的接口后，服务端返回错误信息：该账号已存在。我想要将服务端返回的提示设置到表单中。

```HTML
 <Form bordered form={this.form} onAutoSubmit={(value) => this.handle(value)}>
</Form>
```

```TypeScript
handle = () => {
    axios.post().then(res => {
        // 成功
    }).catch(error => {
       // 失败，将 error 中的错误信息设置到表单中
    });
}
```

**Comment by janryWang** [comment]
第一个问题，使用 https://www.npmjs.com/package/es6-promise-debounce
第二个问题，拿到错误之后使用,form.query("*").forEach((field)=>{
   field.setFeedback(...)
})

**Reply by oswayne** [reply]
感谢🙏
